docs/ch12-qtquick3d/assets.md

```diff
@@ -0,0 +1,106 @@
+# Working with Assets
+
+When working with 3D scenes, the built in meshes quickly grow old. Instead, a good flow from a modelling tool into QML is needed. Qt Quick 3D comes with the Balsam asset import tool, which is used to convert common asset formats into a Qt Quick 3D friendly format.
+
+The purpose of Balsam is to make it easy to take assets created in common tools such as [Blender](https://www.blender.org/), Maya or 3ds Max and use them from Qt Quick 3D. Balsam supports the following asset types:
+
+- COLLADA (``*.dae``)
+- FBX (``*.fbx``)
+- GLTF2 (``*.gltf``, ``*.glb``)
+- STL (``*.stl``)
+- Wavefront (``*.obj``)
+
+For some format, texture assets can also be exported into a Qt Quick 3D-friendly format, as long as Qt Quick 3D supports the given asset.
+
+::: tip Tip
+Balsam also supports converting Qt 3D Studio projects (``*.uia``, ``*.uip``) to Qt Quick 3D. As the feature set is not identical, only the features supported by Qt Quick 3D can be converted.
+:::
+
+## Blender
+
+To generate an asset that we can import, we will use Blender to create a scene with a monkey head in it. We will then export this as a [COLLADA](https://en.wikipedia.org/wiki/COLLADA) file to be able to convert it to a Qt Quick 3D friendly file format using Balsam.
+
+Blender is available from [https://www.blender.org/](https://www.blender.org/), and mastering Blender is a topic for another book, so we will do the most basic thing possible. Remove the original cube, add a mesh, and select to add a monkey. There are a number of video tutorials demonstrating how to do this. The resulting Blender user interface with the monkey head scene can be seen below.
```

Comment:
a + x + enter, shift + a, Mesh->Monkey
It's easy enough to do that maybe that could be suggested inline with the correct formatting, but this is more of a suggestion than a necessity for review.

Reply:
That is a good point. And the keyboard shortcuts seems to change less often than the GUI. Taken!